api/api_ethereum.go

```diff
@@ -897,7 +897,9 @@ func (api *EthereumAPI) GetRawTransactionByHash(ctx context.Context, hash common
 	if err != nil {
 		return nil, err
 	}
-
+	if rawTx[0] == byte(types.EthereumTxTypeEnvelope) {
+		return rawTx[1:], nil
+	}
```

Comment:
RLP-encoded value of Raw Transaction can start with `0x78` by chance, with `0x78` prefix not attached.
Regardless of this PR, are there any potential issues when handle these RawTxs?

Reply:
There will be no chance where raw tx can start with `0x78` except for the Ethereum tx type.
When creating raw tx, `TxInternalDataSerializer.EncodeRLP` is called.
```go
func (serializer *TxInternalDataSerializer) EncodeRLP(w io.Writer) error {
	// if it is the original transaction, do not encode type.
	if serializer.txType == TxTypeLegacyTransaction {
		return rlp.Encode(w, serializer.tx)
	}

	if serializer.txType.IsEthTypedTransaction() {
		ethType := byte(serializer.txType)
		if _, err := w.Write([]byte{byte(EthereumTxTypeEnvelope), ethType}); err != nil {
			return err
		}
	} else {
		if err := rlp.Encode(w, serializer.txType); err != nil {
			return err
		}
	}

	return rlp.Encode(w, serializer.tx)
}
```
What we should take care is only the case `TxTypeLegacyTransaction` because
* if target tx is Ethereum transaction type, the first byte of raw tx (RLP encoded tx) always starts with '0x78'.
* If target tx is Klaytn transaction types(ValueTransfer, ValueTransferMemo, etc...), it always starts with klaytn type int at the first byte.

Then, let's start consider encoding mechanism about `TxTypeLegacyTransaction`.
When encoding `TxTypeLegacyTransaction`, Klaytn apply same encoding scheme as Ethereum does.
So in this case, it always start with first byte from `0xc0` because we treat TxTypeLegacyTransaction payload as a list when RLP encoding for Legacy tx.